packages/react-app/src/components/giveaways/Card.tsx

```diff
@@ -104,36 +131,50 @@ const GiveawayCard = ({
             : 'white',
       }}
     >
-      <CardMedia
-        className="clickable"
-        component="img"
-        height="120"
-        image={
-          giveaway.image
-            ? giveaway.image
-            : '/static/images/placeholder-image.jpg'
-        }
-        onClick={navigateDetails}
-      />
-      <CardContent>
+      <div className="card-media clickable" onClick={navigateDetails}>
+        <img className="img" src={giveaway.image} alt="Giveaway thumb" />
+        {isWinner && (
+          <div className="winner">
+            <div style={{ textAlign: 'center' }}>
+              <img className="icon" src={Trophy} alt="Trophy" />
+              <Typography className="message">You won this contest!</Typography>
+            </div>
+          </div>
+        )}
+      </div>
+      <CardContent className="giveaway-card">
         <Typography
-          className="clickable"
+          className="title text-overflow clickable"
           gutterBottom
           variant="h5"
           onClick={navigateDetails}
-          mt="13px"
         >
           {giveaway.title}
         </Typography>
-        <Typography gutterBottom mt="6px">
+        <Typography className="description text-overflow" gutterBottom>
           {giveaway.description}
         </Typography>
-        <Typography gutterBottom mt="14px">
+        <Typography className="prize text-overflow" gutterBottom>
           <>🏆 {giveaway.prize}</>
         </Typography>
-        <Typography gutterBottom mt="12px">
+        <Typography className="date" gutterBottom>
           <>🗓️ {format(giveaway.endTime, 'MMMM d, yyyy')}</>
         </Typography>
+        {giveaway.winners.length > 0 && (
+          <Typography className="winners" gutterBottom>
+            <>🥳 {getWinnerStr()}</>
```

Comment:
Fragments should only used for wrapping if a component returns multiple elements. They don't add anything inside elements
```suggestion
            🥳 {getWinnerStr()}
```